tests/baseline_test.py

```diff
@@ -3,5 +3,5 @@
 def test_always_passes():
     assert True
 
-# def test_always_fails():
-#     assert False
+def test_always_fails(): # uncommented this test
```

Comment:
How would simply uncommenting this in the live code help this project?  It will, of course, cause the test phase of CI/CD to fail when it runs the tests, and then cause a failure to deploy - even when the code is fine.  There is some value in performing a "known fail" to ensure that failures are detected, but a bit more code is needed for this to work and not simply break the repo deployment cycle.